Code/Framework/AzTest/AzTest/Platform/Android/platform_android_files.cmake

```diff
@@ -8,6 +8,7 @@
 
 set(FILES
     ../Common/UnixLike/AzTest/ColorizedOutput_UnixLike.cpp
+    ../Common/Unimplemented/AzTest/Utils_UnixLike.cpp
```

Comment:
The directory name after "Common" should be "Unix" instead of "Unimplemented". :)